.github/workflows/docker.yml

```diff
@@ -18,7 +18,7 @@ jobs:
       - name: Prepare
         id: prep
         run: |
-          DOCKER_IMAGE=tendermint/tendermint
+          DOCKER_IMAGE=celestiaorg/celestia-core
```

Comment:
Do we actually push that image to somewhere?

Reply:
Nope for now afaik 
A good finding for the issue #554 🧐

Reply:
we won't be pushing anything unless we create a docker hub account and attach it to the celestiaorg github account. Decided to change the docker image name instead of deleting the file, as that seemed useful